pre_commit_hooks/util.py

```diff
@@ -6,7 +6,9 @@
 import sys
 from datetime import datetime
 
-from ruamel import yaml
+import ruamel.yaml
+
+yaml = ruamel.yaml.YAML(typ="safe")
```

Comment:
This fixed an issue one of my team members was having. I'm still confused why they were running into the issue and no one else was though.